### DISCUSSION TITLE
Is it possible to use the built page as a module/component for next page

### DISCUSSION THREAD

**Discussion by sujit-baniya** [illacloud/illa-builder#748] [General]
I stumble with this library and I find it useful. I'm looking for some features for my project. So I was wondering if I could do same using the library.

1) Along with deploying the design to cloud, I could also export as config (JSON)
2) Reuse the designed component as a module or component in another design

**Comment by karminski** [comment]
@wenzhuo2022

**Comment by wenzhuo2022** [comment]
Hello, we will support these two features in future versions. Could you please offer more information about under what circumstances you want to export the design as config?

**Reply by sujit-baniya** [reply]
Use Case:
Once I get the config of the design, I could use it for any purpose: embed on any UI, app using config to display the content and many more.

**Reply by wenzhuo2022** [reply]
Thank you for your reply. We will support it when the components and other features are more complete.

**Reply by sujit-baniya** [reply]
@wenzhuo2022 Thanks for your response. Can you suggest any ETA?
